persistence/src/test/scala/io/vamp/persistence/InMemoryPersistenceActorSpec.scala

```diff
@@ -62,7 +62,7 @@ class InMemoryPersistenceActorSpec extends TestKit(ActorSystem("InMemoryPersiste
           "type" → "in-memory [no persistence]"), "archiving" → true)
       testProbe.send(actor, InfoRequest)
       testProbe.expectMsgPF(30.seconds) {
-        case response: Map[String, Any] ⇒
+        case response: Map[_, _] ⇒
```

Comment:
what was wrong with String, Any and also List[TestArtifact]

Reply:
Because of type erasure (thanks to JVM support of generics), types look good at compile time but at runtime this can cause class cast exception (i.e. not type safe). That is way some IDE's by default raise an warning and here I just "fixing" that warning - you can always do an explicit `asInstanceOf` if you don't care about class cast exception or wrap it with try etc.

Reply:
I didn't get that warning but it is ok I think.